schemas/glean/baseline/baseline.1.parquetmr.txt

```diff
@@ -0,0 +1,163 @@
+message baseline {
+  required group metadata {
+    optional binary EnvVersion (UTF8);
+    optional binary geoSubdivision1 (UTF8);
+    optional binary geoSubdivision2 (UTF8);
+    optional int32 Severity;
+    required binary Host (UTF8);
+    required binary Hostname (UTF8);
+    required binary Logger (UTF8);
+    required binary Type (UTF8);
+    required binary docType (UTF8);
+    required binary documentId (UTF8);
+    required binary geoCity (UTF8);
+    required binary geoCountry (UTF8);
+    required double docVersion;
+    required fixed_len_byte_array(16) Uuid;
+    required int32 Pid;
+    required int64 Timestamp;
+  }
+  required group ping_info {
+    required binary ping_type (UTF8);
+    required binary app_build (UTF8);
+    required binary telemetry_sdk_build (UTF8);
+    required binary client_id (UTF8);
+    required int32 seq;
+    required binary start_time (UTF8);
+    required binary end_time
+  }
+  optional group metrics {
+    optional group boolean (MAP) {
+      repeated group key_value {
+        required binary key (UTF8);
+        optional boolean value;
```

Comment:
These values are all nullable (which it doesn't look like the JSON schema specifies). We can change this to required if we would like.

Reply:
The JSON schema specifies that the values are not nullable, and given the actual implementation of what generates these things, I don't think null values are possible.  That doesn't mean that's right...  What's the use case for sending a metric with a null value?  If there is one, we should probably be explicit about that and bubble that up to the spec document and implementation.

Reply:
I'm not sure there is one. I'll make these required and match the json spec